orsopy/fileio/base.py

```diff
@@ -501,7 +501,9 @@ def _read_header_data(file: Union[TextIO, str], validate: bool = False) -> Tuple
 
         for i, line in enumerate(fi.readlines()):
             if not line.startswith("#"):
-                _ds_lines.append(line)
+                # ignore empty lines
+                if line.strip() != '':
+                    _ds_lines.append(line)
```

Comment:
Would this be better:
```
if not line.startswith("#"):
    _ds_lines.append(line.strip())
```
Alternatively the more Pythonic version of the if statements is:
```
if not line.startswith("#") and line.strip():
    _ds_lines.append(line)
```